ref/app-architecture.md

```diff
@@ -1,32 +1,39 @@
+# App Architecture
 
-Index.php
-    Instantiate DB connection object
-    Pulls breed list (as array of strings) from DB using DB connection object
-    If breed selection given:
-        Give CatHydrator the DB connection object and return the list of CatEntity objects
-        Add main HTML for Top Cat page
-    Translate breed list to HTML inline
-    Translate Cats to HTML inline
-Composer.json
-APIScrape/
-    scrape.php
-    readme.md
-Vendor/ - (directory)
-Entities/ - (directory/Namespace)
-    CatEntity.php - ENTITY CLASS
-        Contains properties:
-            ID
-            img url
-            Breed
-        Contains Getters for above
-    CatEntityTest.php
-Utilities/ - (directory/Namespace)
-    DB.php
-        Class for object that creates connection to Cat database
-        Wrapper object that contains desired database details
-Hydrators/ - (directory/Namespace)
-    CatHydrator.php
-        static function makeListOfCats($db)
-        Takes a database-wrapper and returns an array of CatEntity objects.
+#### Index.php
+- Instantiate DB connection object
+- Pulls breed list (as array of strings) from DB using DB connection object
+- If breed selection given:
+- Give CatHydrator the DB connection object and return the list of CatEntity objects
+    - Add main HTML for Top Cat page
+- Translate breed list to HTML inline
+- Translate Cats to HTML inline
+
+#### Composer.json
+
+## APIScrape/
+#### scrape.php
+#### readme.md
+
+## Vendor/ - (directory)
+
+## Entities/ - (directory/Namespace)
+### CatEntity.php - ENTITY CLASS
+- Contains properties:
+    - ID
+    - img url
+    - Breed
+- Contains Getters for above
+#### CatEntityTest.php
+
+## Utilities/ - (directory/Namespace)
+#### DB.php
+- Class for object that creates connection to Cat database
+- Wrapper object that contains desired database details
```

Comment:
Wrapper object suggests an instantiated object, I think the approach you're going with is instantiating it in index.php and scrape.php and passing it around there.
You could change L31 to: `- Wrapper class that creates connection to Cat database`
And L32 to: `- Contains database connection details`
Does that make sense (is it humanese)?